tests/test_prompt.py

```diff
@@ -47,3 +47,22 @@ def test_invalid_question_type():
                 }
             ]
         )
+
+
+def test_missing_print_message():
+    """Test 'print' raises exception if missing 'message'"""
+    with pytest.raises(PromptParameterException):
+        prompt(
+            [
+                {
+                    "name": "test",
+                    "type": "print",
+                }
+            ]
+        )
+
+
+def test_print_no_name():
+    """'print' type doesn't require a name so it
+    should not throw PromptParameterException"""
+    assert prompt([{"type": "print", "message": "Hello World"}]) == {}
```

Comment:
This test is breaking on Windows. It should work if you import `from prompt_toolkit.output import DummyOutput` and then pass `output=DummyOutput()` to `prompt`. See below for an example.

https://github.com/tmbo/questionary/blob/6643fe006f66802d9e504640f26b9e1a0a9d1253/tests/utils.py#L75